src/models/reach_point.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Base class for modeling a reach point.
+
+Represents a single point on either a reach curve or a reach surface.
+"""
+
+from typing import Iterable
+
+
+class ReachPoint:
+  """A single point on a reach curve."""
+
+  def __init__(self,
+               impressions: Iterable[int],
+               kplus_frequencies: Iterable[int],
+               spend: Iterable[float] = None):
+    """Represents a single point on a reach curve.
+
+    Args:
+      impressions:  The number of impressions that were served by
+        each publisher.  This should be a list
+      kplus_frequencies:  A list of values representing the number
```

Comment:
I would call it kplus_reaches, but kplus_frequencies is also fine.

Reply:
Changed.